CONTRIBUTING.rst

```diff
@@ -0,0 +1,60 @@
+Contributing to Cloudwatch-to-Graphite
+======================================
+
+First off, thanks for taking the time to contribute!
+
+These guidelines are a living document and open to change via a pull
+request.
+
+How Can I Contribute?
+---------------------
+
+Coding style
+~~~~~~~~~~~~
+
+It's a Python project, there is no reason not to follow
+`PEP8 <https://www.python.org/dev/peps/pep-0008/>`__. There are a few exceptions:
+
+* line length < 100 (not 80) if at all possible
+* one-line docstrings are acceptable, but feel free to step up to full docstrings with parameters named and return values specified.
+
+Open Issues
+~~~~~~~~~~~
+
+The project uses GitHub's built-in issue tracker. Check there if there
+is something that matches your skills and interests.
+
+Developing
+~~~~~~~~~~
+
+1. Optional: create a virtual environment:
+   ``virtualenv cw2g && . cw2g/bin/activate``
+2. Install requirements: ``pip install -r requirements.txt``
+3. Run the test suite: ``make test``
+4. Verify the tests pass over all supported Python versions: ``tox``
+
+Pull requests
+~~~~~~~~~~~~~
+
+Standard-issue github project flow, summarized: 1. Fork the report 2.
```

Comment:
typo s/report/repo

This would look better as a list too like:

1. Milk
2. Oreos
3. Ice cream